app/views/admin/document_collection_group_memberships/index.html.erb

```diff
@@ -8,6 +8,19 @@
 <% content_for :context, @collection.title %>
 <% content_for :title_margin_bottom, 4 %>
 
+<% if unavailable_document_count(@group.memberships) > 0 %>
+  <% content_for :banner, render("govuk_publishing_components/components/notice", {
+    title: sanitize("<p>Remove #{unavailable_document_count(@group.memberships)} unavailable document(s) within the group.</p>"),
```

Comment:
We don't need the `<p>` tags here (which I guess means there is no need for `santitize`)